ambry-api/src/main/java/com/github/ambry/frontend/PutBlobMetaInfo.java

```diff
@@ -96,24 +97,34 @@ public List<Pair<String, Long>> getOrderedChunkIdSizeList() {
   }
 
   /**
-   * Serialize a {@link PutBlobMetaInfo} object to a JSON string
+   * Serialize a {@link PutBlobMetaInfo} object to a JSON Object
    * @param metaInfo the {@link PutBlobMetaInfo}
-   * @return the JSON string representation of the object
+   * @return the JSON object
    */
-  public static String serialize(PutBlobMetaInfo metaInfo) {
-    JSONObject rootObject = new JSONObject();
-
+  public static ObjectNode serializeToJasonObject(PutBlobMetaInfo metaInfo) {
```

Comment:
nit: Probably typo in method name for JasonObject?

Reply:
You are right.